src/renderer/containers/archive/entry.js

```diff
@@ -1,11 +1,12 @@
 import { connect } from 'react-redux';
 import { initialize, isDirty } from 'redux-form';
 import Entry from '../../components/archive/entry';
-import { getCurrentEntry, updateEntry, newEntry, deleteEntry, changeMode } from '../../redux/modules/entries';
+import { getCurrentEntry } from '../../../shared/selectors';
+import { updateEntry, newEntry, deleteEntry, changeMode } from '../../../shared/actions/entries';
```

Comment:
Should we start to already look at standardising these paths? If we're using webpack we can just alias the root source directory or `renderer` directory so we can use absolute paths.